src-cpp/app/modules/main/controller.h

```diff
@@ -12,12 +11,9 @@ namespace Modules::Main
 class Controller : public QObject, public IController
 {
 public:
-    explicit Controller(QObject* parent = nullptr);
-    ~Controller() = default;
+    using QObject::QObject;
```

Comment:
sorry but what that means?

Reply:
https://en.cppreference.com/w/cpp/language/using_declaration
see: Inheriting constructors